extensions/sql-database-projects/src/models/project.ts

```diff
@@ -141,6 +141,53 @@ export class Project {
 		return fileEntry;
 	}
 
+	/**
+	 * Set the compat level of the project
+	 * Just used in tests right now, but can be used later if this functionality is added to the UI
+	 * @param compatLevel compat level of project
+	 */
+	public changeDSP(compatLevel: string): void {
+		const newDSP = `${constants.MicrosoftDatatoolsSchemaSqlSql}${compatLevel}${constants.databaseSchemaProvider}`;
+		this.projFileXmlDoc.getElementsByTagName(constants.DSP)[0].childNodes[0].nodeValue = newDSP;
```

Comment:
i am not familiar with the xml schema, is it ok to always use the first DSP and first child node of DSP??

Reply:
added validation to make sure there's only one DSP element tag and it has one child